conf/test.config

```diff
@@ -20,10 +20,75 @@ params {
     max_time   = '6.h'
 
     // Input data
-    // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
 
     // Genome references
-    genome = 'R64-1-1'
+
+    fasta = "https://raw.githubusercontent.com/genomic-medicine-sweden/skierfe/assets/test_data/hg38.test.fa.gz"
+    input = "https://raw.githubusercontent.com/genomic-medicine-sweden/skierfe/assets/samplesheet.csv"
+
+    bed = "https://raw.githubusercontent.com/genomic-medicine-sweden/skierfe/assets/test_data.bed"
+
+    // HiFiCNV
+    hificnv_xy      = "https://raw.githubusercontent.com/genomic-medicine-sweden/skierfe/assets/external/expected_cn.hg38.XY.bed"
+    hificnv_xx      = "https://raw.githubusercontent.com/genomic-medicine-sweden/skierfe/assets/external/expected_cn.hg38.XX.bed"
+    hificnv_exclude = "https://raw.githubusercontent.com/genomic-medicine-sweden/skierfe/assets/test_data/empty.bed"
+
+    // Dipcall
+    dipcall_par = "https://raw.githubusercontent.com/genomic-medicine-sweden/skierfe/assets/external/hs38.PAR.bed"
+
+    // TRGT
+    trgt_repeats = "https://raw.githubusercontent.com/genomic-medicine-sweden/skierfe/assets/external/pathogenic_repeats.hg38.bed"
+
+    // Not tested
+    skip_snv_annotation = true
+    skip_cnv_calling    = true
+
+    parallel_snv = 3 // Create 3 parallel DeepVariant processes
+    preset = "revio"
+
+}
+
+process {
+    withName: '.*:PHASING:WHATSHAP_PHASE' {
+
+        ext.args = '--ignore-read-groups --indels --distrust-genotypes --include-homozygous'
+    }
+
+    withName: '.*:PREPARE_GENOME:MINIMAP2_INDEX'  {
+
+        if(params.preset == 'revio' | params.preset == 'pacbio') {
+            ext.args = "-x map-hifi -I 2G"
+        } else if(params.preset == 'ONT_R9' | params.preset == 'ONT_R10') {
+            ext.args = "-x map-ont -I 2G"
+        }
+    }
+
+    withName: '.*:ASSEMBLY_VARIANT_CALLING:MINIMAP2_INDEX_DIPCALL' {
+
+        ext.args = "-x asm5 -I 2G"
+    }
+
+    withName: '.*:ALIGN_READS:MINIMAP2_ALIGN_UNSPLIT' {
+
+        if(params.preset == 'revio' | params.preset == 'pacbio') {
+            ext.args = "-y -x map-hifi --secondary=no -Y --split-prefix=tmp"
+        } else if(params.preset == 'ONT_R9' | params.preset == 'ONT_R10') {
+            ext.args = "-y -x map-ont --secondary=no -Y --split-prefix=tmp"
+        }
+    }
+
+    withName: '.*:ALIGN_READS:MINIMAP2_ALIGN_SPLIT' {
+
+        if(params.preset == 'revio' | params.preset == 'pacbio') {
+            ext.args = "-y -x map-hifi --secondary=no -Y --split-prefix=tmp"
+        } else if(params.preset == 'ONT_R9' | params.preset == 'ONT_R10') {
+            ext.args = "-y -x map-ont --secondary=no -Y --split-prefix=tmp"
+        }
+    }
+
```

Comment:
Is this exclusively for the test config or should it be used with normal data too?

Reply:
This is only for the test config. Adding `--include-homozygous` allows phasing of a smaller dataset, and the other changes is to minimise the memory requirements by splitting the alignment index. This might not be needed with a small reference genome, and I'll keep working on fine tuning the test data and profile more in the future.